docs/k0sctl-install.md

```diff
@@ -0,0 +1,177 @@
+INFO [ssh] 10.0.0.2:22: downloading k0s 0.11.0 
+INFO [ssh] 10.0.0.1:22: downloading k0s 0.11.0 
+INFO ==> Running phase: Configure K0s    
+WARN [ssh] 10.0.0.1:22: generating default configuration 
+INFO [ssh] 10.0.0.1:22: validating configuration 
+INFO [ssh] 10.0.0.1:22: configuration was changed 
+INFO ==> Running phase: Initialize K0s Cluster 
+INFO [ssh] 10.0.0.1:22: installing k0s controller 
+INFO [ssh] 10.0.0.1:22: waiting for the k0s service to start 
+INFO [ssh] 10.0.0.1:22: waiting for kubernetes api to respond 
+INFO ==> Running phase: Install workers  
+INFO [ssh] 10.0.0.1:22: generating token       
+INFO [ssh] 10.0.0.2:22: writing join token     
+INFO [ssh] 10.0.0.2:22: installing k0s worker  
+INFO [ssh] 10.0.0.2:22: starting service       
+INFO [ssh] 10.0.0.2:22: waiting for node to become ready 
+INFO ==> Running phase: Disconnect from hosts 
+INFO ==> Finished in 2m2s                
+INFO k0s cluster version 0.11.0 is now installed  
+INFO Tip: To access the cluster you can now fetch the admin kubeconfig using: 
+INFO      k0sctl kubeconfig              
+```
+
+And, -- presto your k0s cluster is up and running
+
+Get kubeconfig:
+```sh
+$ k0sctl kubeconfig > kubeconfig
+$ kubectl get pods --kubeconfig kubeconfig -A
+NAMESPACE     NAME                                       READY   STATUS    RESTARTS   AGE
+kube-system   calico-kube-controllers-5f6546844f-w8x27   1/1     Running   0          3m50s
+kube-system   calico-node-vd7lx                          1/1     Running   0          3m44s
+kube-system   coredns-5c98d7d4d8-tmrwv                   1/1     Running   0          4m10s
+kube-system   konnectivity-agent-d9xv2                   1/1     Running   0          3m31s
+kube-system   kube-proxy-xp9r9                           1/1     Running   0          4m4s
+kube-system   metrics-server-6fbcd86f7b-5frtn            1/1     Running   0          3m51s
+```
+
+### Upgrade k0s version on your setup
+
+The previous example installed k0s v0.11.0-beta.2 on one controller and one worker.
+Let's upgrade the cluster to `v0.11.0` 
+**_NOTE:_**  To get the most recent version please visit the release [page](https://github.com/k0sproject/k0s/tags).
+
+The `k0sctl apply` command will determine the running k0s version on each host and perform an upgrade to the target version defined in the configuration file. `k0sctl apply` will then execute the following:
+
+1. Upgrade each controller one-by-one; As long as there’s multiple controllers configured there’s no downtime
+2. Upgrade workers in batches; 10% of the worker nodes are upgraded at a time
+   * Each worker is first drained allowing the workload to “move” to other nodes before the actual upgrade of the worker node components
+   * The process continues after we see the upgraded nodes back in “Ready” state
+   * Drain can be skipped with a --no-drain option
+
+
+In the k0sctl configuration, change the value of `spec.k0s.version`:
+```yaml
+  k0s:
+    version: 0.11.0
+```
+
```

Comment:
```suggestion

When a version has not been specified, k0sctl will check on-line for the latest version and default to using that.

```

Reply:
> Suggested change 
> When a version has not been specified, k0sctl will check on-line for the latest version and default to using that.

small note:
on-line > online

Reply:
Fixed. Looks like I'm still in the 90s :)

> Although a few stodgy editors and style guides still recommend the 1990s-style on-line (with a hyphen) for the computing-related adjective, the trajectory of the language favors online. The latter is now considered acceptable by most dictionaries and English usage guides, and most major publications have changed with the times. Perhaps more important, online is far more common in popular usage.